curite_form/src/main.rs

```diff
@@ -0,0 +1,32 @@
+use clap::Parser;
+use std::fs;
+use warp::Filter;
+use handlebars::Handlebars;
+use serde_json::json;
+
+#[derive(Parser)]
+#[clap(author, version, about, long_about = None)]
+struct Opts {
+    port: u16,
+    path: String,
+}
+
+#[tokio::main]
+async fn main() {
+    let args = Opts::parse();
+
+    let mut hb = Handlebars::new();
+
+    hb.register_template_string(
+        "page",
+        fs::read_to_string(args.path).expect("couldn't read template file"),
+    )
+    .expect("couldn't register template");
```

Comment:
Can we perhaps use [register_template_file](https://docs.rs/handlebars/latest/handlebars/struct.Handlebars.html#method.register_template_file) here?

Reply:
just a note from the docs:
```
Register a template from a path on file system
If dev mode is enabled, the registry will keep reading the template file from file system everytime it’s visited.
```
so it **won't** keep doing disk IO in release builds, which is good